cytotable/convert.py

```diff
@@ -1112,34 +1050,52 @@ def _to_parquet(  # pylint: disable=too-many-arguments, too-many-locals
     # expand the destination path
     expanded_dest_path = _expand_path(path=dest_path)
 
-    # prepare offsets for chunked data export from source tables
-    offsets_prepared = {
+    # check that each source group name has a pagination key
+    for source_group_name in sources.keys():
+        matching_keys = [
+            key for key in page_keys.keys() if key.lower() in source_group_name.lower()
+        ]
+        if not matching_keys:
+            raise CytoTableException(
+                f"No matching key found in page_keys for source_group_name: {source_group_name}"
```

Comment:
Can you add a bit more details to this exception? Any further details about how a user might be able to resolve this exception would be helpful. Why might someone encounter this error?

Reply:
Thanks - I'll add more context here and in the exception. In order for keyset pagination to operate the column / key name must be in the table. This exception raises if the column is not found in the table.